tests/integration/commands/dev/edge-functions.test.ts

```diff
@@ -1,12 +1,38 @@
+import process from 'process'
+
 import { describe, expect, expectTypeOf, test } from 'vitest'
 
 import { FixtureTestContext, setupFixtureTests } from '../../utils/fixture.js'
 import got from '../../utils/got.cjs'
 import { pause } from '../../utils/pause.cjs'
 
+// Skipping tests on Windows because of an issue with the Deno CLI throwing IO
+// errors when running in the CI.
+const isWindows = process.platform === 'win32'
+
+const siteInfo = {
+  account_slug: 'test-account',
+  id: 'foo',
+  name: 'site-name',
+  feature_flags: {
+    edge_functions_npm_support: true,
+  },
+  functions_config: { timeout: 1 },
+}
+
+const routes = [
+  { path: 'sites/foo', response: siteInfo },
+
+  { path: 'sites/foo/service-instances', response: [] },
+  {
+    path: 'accounts',
+    response: [{ slug: siteInfo.account_slug }],
+  },
+]
+
 describe('edge functions', () => {
```

Comment:
Mega nit: Instead of adding `skipIf` to every single test, it should be on the test suite:

```suggestion
describe.skipIf(isWindows)('edge functions', () => {
```